crates/cli/commands/src/test_vectors/mod.rs

```diff
@@ -19,6 +20,18 @@ pub enum Subcommands {
         /// List of table names. Case-sensitive.
         names: Vec<String>,
     },
+    /// Generates test vectors for `Compact` types with `--write`. Reads and checks generated
+    /// vectors with `--read`.
```

Comment:
Unclear what this does, does this read from database?

don't think we need this as a subcommand because this will only cause problems soon